suite-native/app/e2e/pageObjects/accountReceiveActions.ts

```diff
@@ -0,0 +1,26 @@
+import { expect as detoxExpect } from 'detox';
+
+class AccountReceiveActions {
+    async waitForScreen() {
+        await waitFor(element(by.id('@screen/ReceiveAccount'))).toBeVisible();
+    }
+
+    async clickShowAddressButton() {
```

Comment:
We use `tap`, `click` is semantically not correct.
```suggestion
    async tapShowAddressButton() {
```